cmd/operator-sdk/run/cmd.go

```diff
@@ -74,16 +82,24 @@ func NewCmd() *cobra.Command {
 					log.Fatalf("Failed to run operator using OLM: %v", err)
 				}
 			case c.local:
+				// set --watch-namespace flag if the --namespace flag is set
```

Comment:
//TODO: remove namespace flag before 1.0.0

Reply:
i have checked again and add the comment in all relevant locations.